zgw/notificaties-api/src/main/kotlin/com/ritense/notificatiesapi/client/NotificatiesApiClient.kt

```diff
@@ -38,7 +38,7 @@ class NotificatiesApiClient(
 
         return buildNotificatiesWebClient(authentication, baseUrl)
             .post()
-            .uri("/api/v1/abonnement")
+            .uri("abonnement")
```

Comment:
@anna-ritense Fyi

Reply:
I want to discuss this.

Reply:
@Klaas-Ritense I don't think this change should go through. The code is meant to work for /api/v1/, so it should be a part of this line here. It's possible when a v2 of the API will come out, it will not work with the existing code. This class is the client for v1, right?